src/js/lib/api.js

```diff
@@ -0,0 +1,17 @@
+import { buildSearchQuery } from "./search"
+
+export const search = async params => {
+  const body = buildSearchQuery(params)
+
+  const response = await fetch(process.env.SEARCH_API_URL, {
```

Comment:
This will return a max of 10 results, will paging be implemented in a future PR?

Reply:
yeah - I'll add that to the list! :joy: